src/github/taskManager.ts

```diff
@@ -0,0 +1,989 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import Logger from '../common/logger';
+import { ChatSessionWithPR } from './copilotApi';
+import { CopilotRemoteAgentManager } from './copilotRemoteAgent';
+import { FolderRepositoryManager } from './folderRepositoryManager';
+import { RepositoriesManager } from './repositoriesManager';
+
+export interface IssueReference {
+	readonly number: number;
+	readonly nwo?: {
```

Comment:
What does `nwo` mean?

Reply:
Name with owner. It's a term GitHub uses pretty heavily but I'll actually spell it out since it is confusing